Extensions/retags.js

```diff
@@ -27,7 +27,7 @@ XKit.extensions.retags = {
 		var inner, $el;
 		ms.forEach(function(m){
 			$el = $(m.addedNodes);
-			inner = $el.find('.note:not(.ui_post_badge)');
```

Comment:
No, this Mutation Observer runs on every element added to the body, and detects every "note" class. note:not(ui_post_badge) is a way of filtering out the ".note" icon class that tumblr users on the activity page